src/d3-funnel/d3-funnel.js

```diff
@@ -325,10 +331,11 @@ class D3Funnel {
 	 * Create the paths to be used to define the discrete funnel blocks and
 	 * returns the results in an array.
 	 *
-	 * @return {Array}
+	 * @return {Array, Array}
 	 */
 	_makePaths() {
 		const paths = [];
+		const overlayPaths = [];
```

Comment:
`_makePaths` returns two arrays: paths & overlayPaths. All computations happen in one place